client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks/use-patterns-map-by-category.ts

```diff
@@ -6,6 +6,10 @@ const usePatternsMapByCategory = ( patterns: Pattern[], categories: Category[] )
 		const categoriesMap: Record< string, Pattern[] > = {};
 
 		patterns.forEach( ( pattern ) => {
+			// Filter pattern with the meta assembler_waitlist because of rendering issues
+			if ( pattern.pattern_meta.assembler_waitlist ) {
```

Comment:
I guess we can keep these patterns on dev env to help us see whether it looks good or not. WDYT?